examples/cfg/cfg.json

```diff
@@ -1,17 +1,17 @@
 {
-  "control": {},
-  "scheduler": {},
-  "plugins": {
-    "collector": {
-      "cinder": {
-        "all": {
-          "endpoint": "https://public.fuel.local:5000",
-          "user": "admin",
-          "password": "admin"
+  "control": {
+    "plugins": {
+      "collector": {
+        "cinder": {
+          "all": {
+            "endpoint": "http://localhost:5000/v2",
```

Comment:
I agree with @mkuculyma that it should be more descriptive. e.g. http://keystone.endpoint.local:5000.v2. It will avoid miss understandings in future